src/streaming/net/HTTPLoader.js

```diff
@@ -242,6 +243,9 @@ function HTTPLoader(cfg) {
             }
         };
 
+        const ontimeout = function () {
+        };
```

Comment:
Maybe add a debug log here to say it was cancelled because of a timeout? And not because of an error in fetching.